src/main/java/com/google/ehub/data/UserData.java

```diff
@@ -0,0 +1,24 @@
+package com.google.ehub.data;
+
+import java.util.List;
+
+/**
+ * Holds user information used to create JSON object for Profile Servlet's Get request response.
+ */
+public class UserData {
+  private UserProfile profile;
+  private List<String> recommendedUsers;
```

Comment:
these fields can be final

Reply:
I made the fields final in last commit, thanks for pointing that out.